draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -837,12 +837,19 @@ If any step fails, the SD-JWT is not valid, and processing MUST be aborted. Othe
 
 Note that these processing steps do not yield any guarantees to the Holder about having received a complete set of Disclosures. That is, for some digest values in the Issuer-signed JWT (which are not decoy digests) there may be no corresponding Disclosures, for example, if the message from the Issuer was truncated.
 It is up to the Holder how to maintain the mapping between the Disclosures and the plaintext claim values to be able to display them to the user when needed.
+
+
 ## Processing by the Holder  {#holder_verification}
 
 The Issuer MUST provide the Holder an SD-JWT, not an SD-JWT+KB.  If the Holder
 receives an SD-JWT+KB, it MUST be rejected.
 
-For presentation to a Verifier, the Holder MUST perform the following (or equivalent) steps:
+When receiving an SD-JWT, the Holder MUST do the following:
+
+ 1. Process the SD-JWT as defined in (#sd_jwt_verification) to validate the presentation and extract the payload.
```

Comment:
If this means the holder receives it during issuance, then I think presentation is the wrong wording here

Reply:
Correct, that was copy paste from below. 
```suggestion
 1. Process the SD-JWT as defined in (#sd_jwt_verification) to validate the SD-JWT and extract the payload.
```

Reply:
I agree with @paulbastian --- for better or worse the term "presentation" has come to have some specific connotations in the world of digital verifiable credentials that don't align with the holder receiving the thing.

Reply:
```suggestion
 1. Process the SD-JWT as defined in (#sd_jwt_verification) to validate it and extract the payload.
```

Reply:
Either suggestion above would be fine but using "SD-JWT" twice in the one sentence felt a little too redondant.